website/source/docs/providers/aws/r/dynamodb_table.html.markdown

```diff
@@ -38,6 +38,11 @@ resource "aws_dynamodb_table" "basic-dynamodb-table" {
     type = "N"
   }
 
+  ttl {
+    attribute_name = "TimeToExist"
+    enabled = false
+  }
```

Comment:
I was confused by the example attribute name and the way Amazon calls this a Time To Live. In fact the data stored here should be a [unix timestamp](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/howitworks-ttl.html) of when the record expires, not an amount of time (seconds? ms?) for which the data should "live" or "exist". This might be more clear if the `attribute_name` was something along the lines of "ExpirationTimestamp".